tests/x11/toolkits/prepare.pm

```diff
@@ -34,10 +37,7 @@ sub run {
         }
     }
 
-    my $qt5_devel = 'libQt5Core-devel libQt5Gui-devel libQt5Widgets-devel';
-    my $qt6_devel = 'qt6-core-devel qt6-gui-devel qt6-widgets-devel';
-
-    zypper_call "in gcc gcc-c++ tcl tk xmessage fltk-devel motif-devel gtk2-devel gtk3-devel gtk4-devel java java-devel $qt5_devel $qt6_devel";
+    zypper_call "in gcc gcc-c++ tcl tk xmessage fltk-devel motif-devel gtk3-devel gtk3-devel gtk4-devel java java-devel $qt5_devel $qt6_devel";
```

Comment:
gtk2-devel was lost from the list of to be installed packages.

There is a test running that now fails:
https://openqa.opensuse.org/tests/4058664#step/gtk2/11